lib/modules/manager/gomod/artifacts.ts

```diff
@@ -271,7 +273,11 @@ export async function updateArtifacts({
 
     const execCommands: string[] = [];
 
-    let args = 'get -d -t ./...';
+    const goGetDirs =
```

Comment:
Please add a debug log which informs the users what parts have been stripped, else they will be confused what is happening.